google/resource_compute_region_instance_group_manager_test.go

```diff
@@ -798,3 +869,38 @@ resource "google_compute_http_health_check" "zero" {
 }
 	`, template, target, igm, hck)
 }
+
+func testAccRegionInstanceGroupManager_distributionPolicy(template, igm string, zones []string) string {
+	return fmt.Sprintf(`
+resource "google_compute_instance_template" "igm-basic" {
+	name = "%s"
+	machine_type = "n1-standard-1"
+	can_ip_forward = false
+	tags = ["foo", "bar"]
+	disk {
+		source_image = "debian-cloud/debian-8-jessie-v20160803"
```

Comment:
Is it something that's usually hardcoded in tests? At the very least we should use latest and not specific version

Reply:
I just copy/pasted from one of the above tests, to be honest.  It does seem to spin up working instances during the acceptance tests, but I'm happy with updating the `source_image` to something more recent.